settings

```diff
@@ -72,8 +72,10 @@ GIT_PROTOCOL="${GIT_PROTOCOL:-https}"
 GIT_DEVELOP_BRANCH=develop
 GIT_STABLE_BRANCH="${VERSION}-stable"
 GITHUB_NAMESPACE=theforeman
+GITHUB_USER="${GITHUB_USER:-$USER}"
 PACKAGING_DIR="$GIT_DIR/foreman-packaging"
 PACKAGING_GIT_REMOTE=origin
+PACKAGING_FORK_REMOTE="${PACKAGING_FORK_REMOTE:-origin}"
```

Comment:
These both default to origin, but is that really what we want? You'll end up running `gh repo fork theforeman/foreman-packaging --remote-name "origin"` but there is already an origin. Doesn't it make more sense to default to the GH username?
```suggestion
PACKAGING_FORK_REMOTE="${PACKAGING_FORK_REMOTE:-${GITHUB_USER}}"
```